doc/notes/8415_relate_int.md

```diff
@@ -0,0 +1,146 @@
+---
+
+This work is licensed under the Creative Commons CC0 License
+
+---
+
+# Use Relate and Unrelate in MC-3020
+### xtUML Project Implementation Note
+
+### 1. Abstract
+`pyrsl` has been marginally supporting `relate` and `unrelate` statements
+for some time.  Recently, a change in architecture in `pyxtuml` may offer
+speedup.  It is time to begin using the relate and unrelate statements.
+
+### 2. Document References
+[1] [8415](https://support.onefact.net/issues/8415) Use Relate and Unrelate  
+[2] [pyrsl and pyxtuml](https://github.com/xtuml/pyrsl) Python-based Generator  
+
+### 3. Background
```

Comment:
Please note somewhere in the background section the difference between version 0.6.0 and the currently shipped generator. Without that explanation it is confusing why you have three different versions of generator to test against